src/sentry/integrations/github/client.py

```diff
@@ -504,8 +504,6 @@ def get_repos(self, fetch_max_pages: bool = False) -> list[dict[str, Any]]:
         It uses page_size from the base class to specify how many items per page.
         The upper bound of requests is controlled with self.page_number_limit to prevent infinite requests.
         """
-        # XXX: In order to speed up this function we could use ThreadPoolExecutor
-        # to fetch repositories in parallel. See src/sentry/utils/snuba.py
```

Comment:
This is already mentioned inside of `get_with_pagination`
<img width="509" alt="image" src="https://github.com/user-attachments/assets/bef812a8-c99c-4a53-91d9-a5b6710379f7" />